src/output.cpp

```diff
@@ -862,13 +861,23 @@ std::string format_item_info( const std::vector<iteminfo> &vItemDisplay,
                         }
                     }
                 }
-                buffer << sPlus << "<color_" << string_from_color( thisColor ) << ">";
-                if( vItemDisplay[i].is_int ) {
-                    buffer << string_format( "%.0f", vItemDisplay[i].dValue );
+                const char *fmt;
```

Comment:
Please use `std::string`,the code is written as C++ and not C.

Reply:
I'm not sure I agree, but it doesn't matter; this code is unnecessary anyway and I've now removed it.